bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -49,6 +49,12 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     Returns:
         dict: Dict with column names as keys and 1D numpy arrays with content as values
     """
+    # Sanitize input arguments
+    if isinstance(columns, str):
```

Comment:
```suggestion
    if not isinstance(columns, list):
```
If we want to check that the user passes a list I'd say this is better.

Reply:
I thought about this but in python a lot of objects can be iterable. Also, a list is not a tuple, just for example see here:

```python
>>> isinstance([], list)
True
>>> isinstance((), list)
False
```